src/SDK/MonitoringApplianceEmulator/Models/AuthenticationServices.cs

```diff
@@ -84,9 +84,9 @@ public void AuthenticateUser()
                 this.resourceId,
                 this.clientId,
                 this.redirectUri,
-                PromptBehavior.Auto,
+                PromptBehavior.Auto); /*,
                 UserIdentifier.AnyUser,
-                "prompt=consent");
+                "prompt=consent");*/
```

Comment:
>"prompt=consent");*/ [](start = 16, length = 20)

We need to remember to remove this comment once the authentication issue is resolved

Reply:
good catch! reverted

---
In reply to: [231003761](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/13#discussion_r231003761) [](ancestors = 231003761)